src/test/scala/cognite/spark/v1/FilesRelationTest.scala

```diff
@@ -72,9 +70,9 @@ class FilesRelationTest extends FlatSpec with Matchers with SparkTest {
       .write
       .insertInto("filesSource")
 
-    val dfWithTestDirectory = retryWhile[Array[Row]](
-      spark.sql(s"select * from filesSource where assetIds = Array(1739842716040355)").collect,
-      rows => rows.length < 10)
+    val dfWithTestDirectory = retryWhile[List[File]](
+      writeClient.files.filter(FilesFilter(assetIds = Some(Seq(1739842716040355L)))).compile.toList,
+      files => files.length != 10)
```

Comment:
This will get stuck for a while if someone else is creating files like this. Then it will fail on the next line anyway. `< 10` is better imo (though of course it will get stuck if someone is deleting files, but that seems to be more rare)

Reply:
Agreed, will fix this in #373 and rebase when that's merged.